src/sass/letters.scss

```diff
@@ -35,6 +37,15 @@
     padding-bottom: 1.5rem;
   }
 
+  .download-button {
+    margin: 1.5em 0 1em;
+
+    button:disabled {
```

Comment:
In this specific case, it may not matter much. But nesting selectors like this in general will lead to a codebase that's really hard to modify and understand (it's already causing problems in ours, like with the accordion styles). It's better to avoid it if at all possible, especially when using element names (vs classes).